Update Agents section (#1455)
base/docs: docs/ai-agents/quickstart.mdx

    </Note>

    **Claude Desktop config file** — if you prefer to configure manually, open **Settings → Developer → Edit Config** and add:

    | Platform | Config path |
    |----------|-------------|
    | macOS | `~/Library/Application Support/Claude/claude_desktop_config.json` |
    | Windows | `%APPDATA%\Claude\claude_desktop_config.json` |
    | Linux | `~/.config/Claude/claude_desktop_config.json` |

    ```json claude_desktop_config.json
    {
      "mcpServers": {
        "base-account": {
          "url": "https://mcp.base.org"
        }
      }
    }
    ```

    Save and restart Claude Desktop. A wallet icon will appear in the toolbar when connected.
  </Tab>
  <Tab title="ChatGPT">
    <a href="https://chatgpt.com/#settings/Connectors">
      <img src="https://img.shields.io/badge/Add_to_ChatGPT-74AA9C?logo=openai&logoColor=white&style=flat-square" alt="Add to ChatGPT" />
    </a>

    Then:

    1. Enable **Developer Mode** if prompted (under Advanced)
    2. Click **Add custom connector**
    3. Enter the server URL:

    ```text
    https://mcp.base.org
    ```

    4. Save. ChatGPT will prompt you to authorize when you first use a wallet tool.

    <Note>

>       MCP connectors require a ChatGPT Plus or Pro plan.

Reviewer: is this true? I've been using it with my free plan

    </Note>
  </Tab>
  <Tab title="Claude Code">
    Run this in your terminal to add the server to the current project:

    ```bash Terminal
    claude mcp add --transport http base-account https://mcp.base.org
    ```

    To install globally (available across all your projects):

    ```bash Terminal

>     claude mcp add --transport http --scope user base-account https://mcp.base.org

Reviewer: should this be

```
 claude mcp add --transport http --scope user base https://mcp.base.org
```

base-account -> base

    ```

    Verify it connected:

    ```bash Terminal
    claude mcp list
    ```

>     The `base-account` server will show with a tool count once active. You can also run `/mcp` inside a Claude Code session to see server status.

Reviewer: ```suggestion
    The `base` server will show with a tool count once active. You can also run `/mcp` inside a Claude Code session to see server status.
```

  </Tab>
  <Tab title="Codex">
    ```bash Terminal

>     codex mcp add base-account --url https://mcp.base.org/

Reviewer: ```suggestion
    codex mcp add base --url https://mcp.base.org/
```

    ```

    Or add to your `codex.toml`:

    ```toml codex.toml

>     [mcp_servers.base-account]

Reviewer: ```suggestion
    [mcp_servers.base]
```

    url = "https://mcp.base.org/"
    ```
  </Tab>
  <Tab title="Cursor">
    <a href="cursor://anysphere.cursor-deeplink/mcp/install?name=base-account&config=eyJ1cmwiOiJodHRwczovL21jcC5iYXNlLm9yZyJ9">
      <img src="https://img.shields.io/badge/Add_to_Cursor-000000?logo=cursor&logoColor=white&style=flat-square" alt="Add to Cursor" />
    </a>

    Or add manually to `~/.cursor/mcp.json` (global) or `.cursor/mcp.json` (project):

    ```json mcp.json
    {
      "mcpServers": {

>         "base-account": {

Reviewer: ```suggestion
        "base": {
```

          "url": "https://mcp.base.org"
        }
      }
    }
    ```

    Restart Cursor, then open **Settings → MCP** to confirm `base-account` shows as active.
  </Tab>
  <Tab title="VS Code">
    <a href="https://vscode.dev/redirect/mcp/install?name=base-account&config=%7B%22type%22%3A%22http%22%2C%22url%22%3A%22https%3A%2F%2Fmcp.base.org%22%7D">
      <img src="https://img.shields.io/badge/Add_to_VS_Code-007ACC?logo=visualstudiocode&logoColor=white&style=flat-square" alt="Add to VS Code" />
    </a>

    Or add manually to `.vscode/mcp.json` (workspace) or your user MCP settings:

    ```json .vscode/mcp.json
    {
      "servers": {

>         "base-account": {

Reviewer: ```suggestion
        "base": {
```

          "type": "http",
          "url": "https://mcp.base.org"
        }
      }
    }
    ```

    Open **Copilot Chat** in agent mode and confirm the wallet tools appear in the tool picker.
  </Tab>
  <Tab title="Windsurf">
    Add to `~/.codeium/windsurf/mcp_config.json`:

    ```json mcp_config.json
    {
      "mcpServers": {

>         "base-account": {

Reviewer: ```suggestion
        "base": {
```

          "url": "https://mcp.base.org"
        }
      }
    }
    ```

    Restart Windsurf. Open **Cascade** — the MCP tools will be available once connected.

    <Tip>
      You can also add MCP servers from the UI: open **Cascade** → click the **hammer icon** → **Add MCP Server**.
    </Tip>
  </Tab>
</Tabs>

<Note>

>   **Other clients** — Any MCP client that supports remote HTTP (Streamable HTTP) servers works with `https://mcp.base.org`. See the [setup guide](/ai-agents/setup) for Zed, JetBrains, Le Chat (Mistral), Warp, and others. On mobile, the Claude connector above works in the iOS and Android apps.

Reviewer: this would be better as it's own tab IMO, removes unnecessary instruction for most ppl

</Note>

## Step 2 — Authorize

> The first time you use a wallet tool, your assistant will redirect you to `keys.coinbase.com` to connect your Base Account. You'll need a Coinbase account. Once authorized, all transactions still require individual approval at that same URL.

Reviewer: ```suggestion
The first time you use a wallet tool, your assistant will redirect you to `keys.coinbase.com` to connect your existing Base Account or to create one. Once authorized, all transactions still require individual approval at that same URL.
```

## Step 3 — Try it

Ask your assistant:

```text
Show me my wallets
```

```text
What's my USDC balance on Base?
```

```text
Send 1 USDC to vitalik.base.eth
```

Every send, swap, or sign operation will give you an approval link. Open it, review the transaction, and confirm.

---

## Add a plugin — Morpho lending

Morpho lets your assistant query vaults, check APYs, and execute lending operations on Base — then sign them through your Base Account.

Add both MCPs to your config:

<Tabs>
  <Tab title="Claude">
    ```json claude_desktop_config.json
    {
      "mcpServers": {
        "base-account": {
          "url": "https://mcp.base.org"
        },
        "morpho": {
          "url": "https://mcp.morpho.org/"
        }
      }

>     }

Reviewer: I think the moonwell skill would be better example for the first page. It's lower lift to go from setting up our MCP to a "magical" experience.

    ```
  </Tab>
  <Tab title="Claude Code">
    ```bash Terminal
    claude mcp add --transport http base-account https://mcp.base.org
    claude mcp add morpho --transport http https://mcp.morpho.org/
    ```
  </Tab>
  <Tab title="Cursor / VS Code / Windsurf / Codex">
    Add a second entry alongside `base-account`:

    ```json
    "morpho": { "url": "https://mcp.morpho.org/" }
    ```
  </Tab>
</Tabs>

Then try:

```text
Find the best USDC vault on Base by APY and deposit 100 USDC
```

Morpho prepares the transaction; your Base Account signs it. See [Morpho plugin →](/ai-agents/plugins/morpho)

> 

Reviewer: call to action here to try skills
